docs/cs-manual/source/record-filters.rst

```diff
@@ -0,0 +1,277 @@
+       should use a string for those.
+
+       
+``value(<value>, (<lower>, <upper>))``
+   Match a value type, such as a number, with a range of possible values. The
+   argument ``<value>`` is the string name of the value we want to compare, the
+   second argument is a pair of lower and upper bounds for the range interval.
+
+   One can use either parentheses ``()`` or brackets ``[]`` to surround the
+   pair. Brackets are inclusive and parentheses are exclusive. We can also mix
+   both styles:
+
+   .. code::
+
+      value("Weight", [20, 50))
+
+   This example will match any objects with a weight greater or equal to 20
+   and strictly less than 50.
+
+
+Logical Expressions
+-------------------
+
+``not <expression>``
+   Logically negates the result of an expression. If ``<expression>`` evaluates
+   to ``true`` the negation is ``false``, and if ``<expression>`` evaluates to
+   ``false`` the negation is ``true``. Note that there are no parentheses
+   around the argument.
+
+``or(<expr1>, <expr2>, ..., <exprN>)``
+   Logical disjunction, evaluates to ``true`` if at least one argument
+   evaluates to ``true`` as well, otherwise the expression evaluates to
+   ``false``.
+
+   As an example assume we want to filter for both NPCs and creatures; the
+   expression for that use-case is
+
+   .. code::
+      
+      or(string("record type", "npc"), string("record type", "creature"))
+
+   In this particular case only one argument can evaluate to ``true``, but one
+   can write expressions where multiple arguments can be ``true`` at a time.
+
+``or(<expr1>, <expr2>, ..., <exprN>)``
+   Logical conjunction, evaluates to ``true`` if and only if all arguments
+   evaluate to ``true`` as well, otherwise the expression evaluates to
+   ``false``.
+
+   As an example assume we want to filter for weapons weighting less than a hundred
+   units The expression for that use-case is
+
+   .. code::
+      
+      and(string("record type", "weapon"), value("weight", (0, 100)))
+
+
+
+Creating and saving filters
+***************************
+
+In order to create and save a new filter go to the filters table, right click
+and select the option *Add Record* from the context menu. You are given a
+choice between project- or session scope. Choose the scope from the dropdown
+and type in your desired ID for the filter. A newly created filter does nothing
+since it still lacks expressions. In order to add your queries you have to edit
+the filter record.
+
+
+Replacing the default filters set
+=================================
+
+OpenMW CS allows you to substitute the default filters set provided by the
+developers your own filters. In order to do so you have to create a new
+project, add the desired filters, remove the undesired ones and save. Rename
+the file to ``defaultfilters`` (remove the ``.omwaddon.project`` file
+extension) and place it inside your configuration directory.
```

Comment:
Also this mixes things up. You can replace the project filters as described before inside OpenMW-CS. Replacing the defaultfilters file is what would actually change the defaults for all content files that have not been edited on this computer/user-account before. Since this manual is only about the basics, maybe that should be taken out since it is an advanced task?